### DISCUSSION TITLE
zh/docs/master/deploy/docker

### DISCUSSION THREAD

**Discussion by giscus** [gorse-io/docs#111] [Giscus]
# zh/docs/master/deploy/docker

Docker 部署 用Docker Compose或Docker Swarm管理小规模微服务集群是不错的选择。Gorse官方提供了Docker镜像： Docker镜像 版本 镜像大小 拉取次数 gorse-master gorse-server gorse-worker gorse-in-one

https://gorse.io/zh/docs/master/deploy/docker.html

**Comment by dmzlingyin** [comment]
ports:
      - 8086:8086   # gRPC port
      - 8088:8088   # HTTP port

这两个写反了吧
